scripts/generate_rust_target.rs

```diff
@@ -152,7 +152,6 @@ fn main() {
             "data-layout",
             "e-m:e-p:32:32-Fi8-i64:64-v128:64:128-a:0:32-n32-S64",
         );
-        ts.push("dynamic-linking", true);
```

Comment:
Should probably be set to false instead.

Reply:
This is the same case as `executables`, i.e. it defaults to false.